compass:
    px: 200
    py: 50
    d: 300
artificial_horizon:
    px: 1000
    py: 50
    d:
height:
    px: 1000
    py: 500
linear_velocity:
    px: 1000
    py: 250